sympy/physics/quantum/state.py

```diff
@@ -909,6 +910,50 @@ def expr(self):
         """
         return self._args[0]
 
+    @staticmethod
+    def extract_expression_from_wavefunction(input_expression):
```

Comment:
This function does not seem to be used anywhere.

Reply:
I added this as a handicap to extract the expression from the wavefunction objects. In the current state, simply doing `.expr` on a linear combination of wavefunction objects yields an error.